version

```diff
@@ -0,0 +1 @@
+1.0.0-beta2
```

Comment:
needs updating (later?)

Reply:
Yes, let's do straight backports and update this file after